charts/opentelemetry-collector/values.schema.json

```diff
@@ -474,6 +474,21 @@
         },
         "annotations": {
           "type": "object"
+        },
+        "additionalIngresses": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "additionalProperties": true,
```

Comment:
Let's define the full schema here and disallow `additionalProperties`

Reply:
I've defined the full schema for both root ingress and additional ingresses as both are using the same schema (except `enabled` and `name` being in each respectively). I've also set required properties based on [Kubernetes reference](https://kubernetes.io/docs/reference/kubernetes-api/service-resources/ingress-v1/) (basically the path properties) and set enum for path type.